test/MarketMaker.js

```diff
@@ -265,19 +276,19 @@ contract('marketMaker', function(accounts) {
     ];
 
     let getMaxClaimableEth = function(state) {
-      if (state.month >= periods) {
-        help.debug("calculating maxClaimableEth with", startingMMBalance, state.totalClaimedEth,
+      if (state.marketMakerMonth >= periods) {
+        help.debug("calculating maxClaimableEth with", startingMMBalance, state.marketMakerClaimedEth,
           state.returnedWeiForBurnedTokens);
         return startingMMBalance.
-          minus(state.totalClaimedEth).
+          minus(state.marketMakerClaimedEth).
           minus(state.returnedWeiForBurnedTokens);
       } else {
         const totalSupplyWei = web3.toWei(tokenTotalSupply, 'ether');
         const maxClaimable = startingMMBalance.
           mul(state.claimablePercentage).dividedBy(priceFactor).
-          mul(totalSupplyWei - state.burnedTokens).
+          mul(totalSupplyWei - state.marketMakerBurnedTokens).
```

Comment:
`.minus` form BigNumber can be used here

Reply:
good catch, fixed in 2e49ff2

Reply:
I changed it back to use `-` instead of `.minus`, because `totalSupplyWei` was not a bigNumber so the test was failing... we could make it a BigNumber, but in reality we know it is small enough that we can keep it as is (it's 100 lifWei minus the burnedtokens which is also a relatively small number)